easybuild/easyconfigs/c/CondaCreate/condacreate-1.0.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_CondaCreate'
+
+name = "condacreate-env"
+version = "1.0"
+variant = "Linux-x86_64"
```

Comment:
yes, because `versionsuffix` means something in EasyBuild, while `variant` is just a local variable in the easyconfig

You'll need to adjust the easyconfig filename accordingly to make the test suite happy.